docs/hosting/configuration/configuration-examples/custom-certificate-authority.md

```diff
@@ -0,0 +1,41 @@
+---
+#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
+title: Configure n8n to use your own certificate authority
+description: Customize the n8n container to work with self signed certificates when connecting to services.
+contentType: howto
+---
+
+# Configure n8n to use your own certificate authority or self signed certificate
+
+Since version 1.42.0 it has been possible to add your own certificate authority or self signed certificate to n8n, This means you are able to trust a certain certificate instead of trusting all invalid certificates which is a potential security risk.
```

Comment:
```suggestion
Since version 1.42.0 it has been possible to add your own certificate authority or self signed certificate to n8n. This means you are able to trust a certain certificate instead of trusting all invalid certificates, which is a potential security risk.
```